pkg/kernel/networkservice/applyiptables4nattemplate/common.go

```diff
@@ -0,0 +1,123 @@
+// Copyright (c) 2022 Xored Software Inc and others.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build linux
+// +build linux
+
+package applyiptables4nattemplate
```

Comment:
```suggestion
package applyiptables
```
or
```suggestion
package iptables
```

I think could be better.